packages/manager/apps/web/client/app/app.js

```diff
@@ -547,4 +550,6 @@ angular
         }
       });
     },
-  );
+  )
+  .service('hostingEmailService', hostingEmailService)
```

Comment:
Shouldn't be located here.

Maybe could be moved to `packages/manager/web/client/app/hosting/email`?